GrailDiary/GrailDiary/Models/POI.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  POI.swift
+//  GrailDiary
+//
+//  Created by Mike Nichols on 5/14/20.
+//  Copyright © 2020 Mike Nichols. All rights reserved.
+//
+
+import Foundation
+
+struct POI {
+    
+    location: String
+    country: String
+    clues: [String]
+    
+}
+
+var poiList: [POI] = []
```

Comment:
Not sure why this is here.

Reply:
seemed like a good place to create a global array of POIs to access around the app.